packages/iov-bns/src/decode.spec.ts

```diff
@@ -449,6 +450,21 @@ describe("Decode", () => {
       });
     });
 
+    it("works for TransferUsernameTx", () => {
+      const transactionMessage: codecImpl.bnsd.ITx = {
+        usernameTransferTokenMsg: {
+          username: "bobby",
```

Comment:
namespace is now mandatory in the `username` field, which should be visible in the test if possible